dissect/target/plugins/general/loaders.py

```diff
@@ -14,10 +12,8 @@ def check_compatible(self):
     def loaders(self):
         """List the available loaders."""
 
-        loaders = itertools.chain(LOADERS, [DirLoader])
-        loader_names = [loader.attr for loader in loaders]
-        loader_names.sort()
+        loader_names = sorted(getattr(loader, "attr", loader.__name__) for loader in LOADERS + [DirLoader])
```

Comment:
As the `attr` property is part of the "lazy loading" properties of these modules, using it is dubious to begin with. So replacing it all together with the property that is actually meant is not a problem:

```suggestion
        loader_names = sorted(loader.__name__ for loader in LOADERS + [DirLoader])
```

Note that the code on HEAD has already changed significantly, so this change will need some reworking.